<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>1141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="editIp">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonConnect">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>170</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>731</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="editInput">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>731</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="editDisplay">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>610</y>
     <width>731</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSend">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>1020</y>
     <width>170</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editName">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="editPassword">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSignIn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>170</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>SignIn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSignUp">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>80</y>
     <width>170</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>SignUp</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
